app/views/dashboard/index.html.erb

```diff
@@ -2,11 +2,7 @@
 <%= link_to "My items", merchant_items_path(@merchant.id) %>
 <%= link_to "My invoices", merchant_invoices_path(@merchant.id) %>
 
-<% @merchant.top_5_customers.each do |customer| %>
-  <div id="customer<%= customer.id %>">
-  <p><%= customer.first_name %> <%= customer.last_name %> <%= customer.transaction_count %></p>
-  </div>
-<% end %>
+<%= render partial: '/layouts/partials/top_customers' %>
```

Comment:
Wondering about the partial here, are we going to  need it anywhere else?

Reply:
I was thinking our view would look more clean if we render partial these big methods. Thoughts?

Reply:
Sounds great to me!

Reply:
Yeah, I think this sounds like a great idea!

Reply:
Agreed.